src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php

```diff
@@ -22,12 +22,12 @@
 abstract class WebTestCase extends KernelTestCase
 {
     /**
-     * Creates a Client.
+     * Creates a KernelBrowser.
      *
      * @param array $options An array of options to pass to the createKernel method
      * @param array $server  An array of server parameters
      *
-     * @return Client A Client instance
+     * @return KernelBrowser A KernelBrowser instance
      */
     protected static function createClient(array $options = [], array $server = [])
```

Comment:
should we rename that method too ?

Reply:
I thought about renaming methods but I think that's too much. What others think?

Reply:
With a hard copy pasting you could also add return type hints on methods.